core/primitives/src/types.rs

```diff
@@ -536,3 +537,19 @@ pub enum ValidatorKickoutReason {
 
 #[derive(PartialEq, Eq, Clone, Debug, BorshSerialize, BorshDeserialize, Serialize)]
 pub struct StateHeaderKey(pub ShardId, pub CryptoHash);
+
+/// Provides information about current epoch validators.
+/// Used to break dependency between epoch manager and runtime.
+pub trait EpochInfoProvider {
+    /// Get current stake of a validator in the given epoch.
+    /// If the account is not a validator, returns `None`.
+    fn get_validator_stake(
```

Comment:
Just `fn validator_stake` for consistency with https://github.com/nearprotocol/nearcore/pull/2504/files#diff-d40cb4d4c971b522c3125d3149574eefR520?

Also, rust style guideline suggests to not use `get_` prefix for getters https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter